.gitignore

```diff
@@ -5,3 +5,4 @@ Data/Scripts/GardenConquest/GardenConquest.v11.suo
 modinfo.sbmi
 GardenConquestTests/bin/
 GardenConquestTests/obj/
+*.bak
```

Comment:
Which backups are you referring to? So with git, you never need to save backups of files that have already been committed to the tree, because they're always available through the commit history. You can check out any version of any file from any time it's been committed. 

The gitignore is meant to include all files that should everyone working on this project would want to ignore - so we only include file types that are directly correlated with the body of code. Personal preferences we leave to the global .gitignore file (likely already available in your ~ directory). 

This isn't to be a dick, it's just long term better practice to be semantic about what we include to help describe the project and ensure no one is ever surprised because they try to add a legitimate file but it's ignore due to an unfamiliar ignore.